main.cpp

```diff
@@ -20,6 +21,11 @@ extern "C" void TIMER1_Handler(void);
 
 int main(void)
 {
+#if MBED_CONF_APP_WAIT_FOR_SYNC
+    tfm_log_printf("Waiting for Greentea host\n");
+    GREENTEA_SETUP(60, "default_auto");
```

Comment:
The first argument is the timeout, and "default_auto" is the name we normally use when there's no host interaction required during the test (apart from the sync signal at the start)